R/create_dap_specific_concept.R

```diff
@@ -83,6 +84,19 @@ create_dap_specific_concept <- function(codelist, data_db, name_attachment, save
     cols_temp <- na.omit(as.character(cols[j]))
     values_temp <- toupper(na.omit(as.character(values[j])))
     value <- codelist[[j, "keep_value_column_name"]]
+    if(add_meaning == TRUE){
+      columns_db_table <- dbListFields(save_db, name_edited)
+      meaning_column_name <- columns_db_table[str_detect(columns_db_table,'meaning')]
```

Comment:
`str_detect` is a `stringr` function, right?

Reply:
Indeed, added in the last commit.